R/page.R

```diff
@@ -161,16 +252,59 @@ page_sidebar <- function(..., sidebar = NULL, title = NULL, fillable = TRUE, fil
 }
 
 
-#' @rdname page
-#' @inheritParams navset_bar
-#' @seealso [shiny::navbarPage()]
+#' Multi-page app with a top navigation bar
+#'
+#' @description
+#' `page_navbar()` provides a full-screen app with a navigation bar that behaves
+#' like a multi-page app (but is really a single-page app).
+#'
 #' @param fillable_mobile Whether or not `fillable` pages should fill the viewport's
 #'   height on mobile devices (i.e., narrow windows).
 #' @param underline Whether or not to add underline styling to page links when
 #'   active or focused.
 #' @param window_title the browser window title. The default value, `NA`, means
 #'   to use any character strings that appear in `title` (if none are found, the
 #'   host URL of the page is displayed by default).
+#' @inheritParams navset_bar
+#' @inheritParams page
+#'
+#' @seealso [shiny::navbarPage()]
```

Comment:
Now that this is on it's own page, I think we should reconsider what goes into the `@seealso`. Maybe this should point to navigation (e.g., `nav_panel()`, `navset_bar()`), components (e.g, `card()`, `value_box()`), and other page layouts (e.g., `page_sidebar()`, `page_fillable()`)?

Reply:
Done in d1b7743bff29aecbbbda823cc50bb083861cd0de